### PR TITLE
Reduced pickelhaube's effectiveness as a melee weapon

### PR DESCRIPTION
**Pull request by Barhandar**
It's nowhere near +1 to-hit bonus because it's short, unwieldy (but can be gripped), and has awkward balance, so the bonus was changed to -3 and might even get to be -4 if it's considered entirely unbalanced.

On the other hand, now the spike is actually sharp!

### CODE REVIEW

**Comment by DavidKeaton** [comment]
good catch

File: data/json/items/armor.json [hunk 1]
```diff
@@ -6411,22 +6411,22 @@
...
         "warmth" : 15,
         "phase" : "solid",
         "environmental_protection" : 0,
         "encumbrance" : 20,
-        "bashing" : 10,
+        "bashing" : 2,
         "coverage" : 70,
         "material_thickness" : 3,
-        "flags" : ["FANCY"]
+        "flags" : ["FANCY", "SPEAR"]
```
Comment (Rivet-the-Zombie) [inline data/json/items/armor.json]: Please replace ````SPEAR```` with ````STAB````. ````SPEAR```` includes specialized combat messages and special modified stick-in rolls that wouldn't be appropriate for this.

Comment (Barhandar) [inline data/json/items/armor.json]: Okay, wasn't aware STAB also changed it to piercing.

**Comment by DavidKeaton** [comment]
Same Type Attack Bonus?

**Comment by Barhandar** [comment]
>Same Type Attack Bonus? :-P

What?

**Comment by DavidKeaton** [comment]
It's from Pokemon.

**Pull request merged**